catkin_ws/src/kpr_arm_control/config/controllers.yaml

```diff
@@ -0,0 +1,11 @@
+controller_list:
+  - name: ""
+    action_ns: follow_joint_trajectory
+    type: FollowJointTrajectory
+    joints:
+      - shoulder_pan_joint
+      - shoulder_lift_joint
+      - elbow_joint
+      - wrist_1_joint
+      - wrist_2_joint
+      - wrist_3_joint
```

Comment:
I'd prefer to leave them in because we might need to adjust them in the future, also the universal robots package calls the folder containing the robot model the config folder, so I don't think there's a strict convention on this.